pkg/awsclient/client.go

```diff
@@ -383,13 +408,14 @@ func newClient(controllerName, awsAccessID, awsAccessSecret, token, region strin
 	}
 
 	return &awsClient{
-		ec2Client:     ec2.New(s),
-		iamClient:     iam.New(s),
-		orgClient:     organizations.New(s),
-		stsClient:     sts.New(s),
-		supportClient: support.New(s),
-		s3Client:      s3.New(s),
-		route53client: route53.New(s),
+		ec2Client:           ec2.New(s),
+		iamClient:           iam.New(s),
+		orgClient:           organizations.New(s),
+		route53client:       route53.New(s),
+		s3Client:            s3.New(s),
+		stsClient:           sts.New(s),
+		supportClient:       support.New(s),
+		serviceQuotasClient: servicequotas.New(s, aws.NewConfig().WithRegion("us-west-2")),
```

Comment:
Why are you specifying a specific region here?

Reply:
That was an oversight on my part.  I was testing whether or not the quotas were region-based and forgot to remove that.